third_party/2and3/requests/sessions.pyi

```diff
@@ -16,6 +16,7 @@ from . import adapters
 from . import status_codes
 
 BaseAdapter = adapters.BaseAdapter
+AuthBase = auth.AuthBase
```

Comment:
That's weird, I would have expected that code to work. I don't have more time to look right now though.